packages/spatial-id-svc-area/src/client-blocked-areas.ts

```diff
@@ -358,6 +371,11 @@ export const getSignalAreas = async function* ({
     payload,
     abortSignal,
   })) {
+    if (chunk.result.objects[0].objectId !== '0') {
```

Comment:
Isnt it better to do optional chaining? using ?. or else if data wasn't present it could lead to errors